src/types/User.ts

```diff
@@ -0,0 +1,5 @@
+export interface User {
+  id: number,
+  name: string,
+  email: string,
```

Comment:
types should end with semicolons
```suggestion
  id: number;
  name: string;
  email: string;
```

Reply:
done